_docs/sandbox-testing.md

```diff
@@ -158,6 +158,152 @@ end
 {::comment}AUTO_GENERATED_CONTENT_END{:/comment}
 
 
+# Testing based on input
+You can test success/error scenarios based on payload attributes such as:
+- mobile_number
+- amount
+
+## XOF mobile collections
+### Success scenario
+When testing collections for `XOF::Mobile`, you can use a mobile number ending with `00` to have a collection succeed. The payload for this scenario would be:
+{% capture data-raw %}
+
+```javascript
+{
+  "id": "1625c534-6db4-4f3a-adf2-62a8bec89080",
+  "type": "XOF::Mobile",
+  "ux_flow": "ussd_voucher",
+  "state": "initial",
+  "in_details": {
+    "otp": "1234",
+    "phone_number": "+221771234700",
+    "mobile_provider": "orange"
+  },
+  "out_details": {
+    "style": "info"
+  },
+  "instructions": {}
+}
+```
+
+{% endcapture%}
+
+{% include language-tabbar.html prefix="collection-xof-mobile-success" raw=data-raw %}
+
+Once the collection request is processed, you'd receive a `payin_method.paid_in` webhook:
+{% capture data-raw %}
+
+```javascript
+{
+  "webhook": "ddef6199-6171-43ba-bbb5-29fb06c9df9f",
+  "event": "payin_method.paid_in",
+  "object": {
+    "id": "1625c534-6db4-4f3a-adf2-62a8bec89080",
+    "type": "XOF::Mobile",
+    "ux_flow": "ussd_voucher",
+    "state": "success",
+    "state_reason_details": {
+      "code": "0",
+      "category": "paid",
+      "messages": [
+        "Success",
+        "Success",
+        "Success"
+      ],
+      "description": "The transaction was successfully completed."
+    },
+    "in_details": {
+      "otp": "1234",
+      "ux_flow": "ussd_voucher",
+      "phone_number": "0771234700",
+      "mobile_provider": "orange"
+    },
+    "out_details": {
+      "style": "info"
+    },
+    "instructions": {
+    },
+    "transaction_id": "0d59466a-dc2f-4aca-bad1-d4a98da37697",
+    "transaction_external_id": "510c4e28-fe33-4c06-bd79-49d52d5b3094"
+  }
+}
+```
+
+{% endcapture%}
+
+{% include language-tabbar.html prefix="collection-xof-mobile-paid-in" raw=data-raw %}
+
+
+### Error scenario
+When testing collections for `XOF::Mobile`, you can use a mobile number ending with `05` to have a collection fail. The payload for this scenario would be:
```

Comment:
this looks wrong. We have a standard that says ending in 0 is always success, ending in 8 or 9 is failure, the rest is configurable. We should bne using the standard across all our harnesses. If this is not the case for PayDunya let's fix the harness

Reply:
I'll have this fixed on the harness.

Reply:
Here's the latest standard we have https://azafinance.atlassian.net/browse/TEM-41 I just seen it mostly refers to payouts but it can likely be converted to support collections in a similar way